src/components/ui-board/content-element/ContentElementBar.vue

```diff
@@ -1,112 +1,106 @@
 <template>
 	<div tabindex="-1" class="content-element-bar">
-		<div v-if="$slots.display">
-			<ContentElementDisplay>
-				<template #display>
-					<slot name="display" />
-				</template>
-
-				<template #menu>
-					<slot name="menu" />
-				</template>
-			</ContentElementDisplay>
+		<div v-if="$slots.menu" class="three-dot-menu">
+			<slot name="menu" />
 		</div>
+
+		<div v-if="$slots.display" class="content-element-display">
+			<slot name="display" />
+		</div>
+
 		<div
-			:class="backgroundClass"
-			class="rounded-b pb-4"
 			v-if="
-				icon ||
-				$slots.logo ||
-				$slots.title ||
-				$slots.menu ||
-				$slots.element ||
-				$slots.subtitle ||
-				$slots.description
+				$slots.title || $slots.element || $slots.subtitle || $slots.description
 			"
+			:class="dynamicClasses"
```

Comment:
Do we need a computed property here or would it be better to inline it?
```suggestion
			:class="{ 'bg-grey-lighten-4': props.hasGreyBackground }"
```

Reply:
the computed does not make sense any longer ✅